app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-Root view
\ No newline at end of file
+<ul>
+  <% @users.each do |user| %>
+    <h4><%= user.id %></h4>
+    <li><h1><%= user.name %></h1></li>
```

Comment:
- [ ] It is also required that clicking on each user take us to the user show page. Your user is not clickable. Kindly fix that.

### Expected show page

![show-page](https://github.com/Gh-Reza/BlogApp/assets/26597432/8ac130f9-912f-493d-bb50-f468e72cb7ae)

### Your Page
[your-app.webm](https://github.com/Gh-Reza/BlogApp/assets/26597432/7b248587-7add-40f6-b94f-659181835153)